canonical/ubuntu.com
# Redeem beta invites

## Done

- Beta participants can redeem their invitation codes

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Generate activation key from the back office tool
- Go to /credentials/shop/redeem?code={activationKey}
- It should attach an exam to the user
## Issue / Card

Fixes https://warthogs.atlassian.net/browse/CRED-230

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
